pkg/apps/application-list.jsx

```diff
@@ -147,6 +172,10 @@ export const ApplicationList = ({ metainfo_db, appProgress, appProgressTitle, ac
                                                action={action} />);
     }
 
+    const data_missing_msg = (dataPackagesInstalled == false && !refresh_progress)
+        ? _("Appstream data package is missing, refresh to install it")
```

Comment:
I think this is needlessly jargony. What about "Application information is out of date." plus a "Refresh" button right next to it?

We should also show this message when the data or config packages have updates available, not just when they are not installed at all.

Reply:
I did this (with slightly different wording), see screenshots in the PR description.

The "have updates available" is precisely what I don't want to do, as it's really expensive, and I don't want to trigger a Refresh automatically.

Reply:
The Refresh might have been triggered externally, maybe via cron or from the Software updates page, no?

Reply:
But, yes, it's fine as it is. Once the data/config package is installed, normal OS hygiene should take care of keeping it up-to-date.